package.json

```
@@ -22,7 +23,7 @@
     "format": "prettier --loglevel warn --write \"**/*.{js,jsx,ts,tsx,css,md,yml,json}\"",
     "dev": "lerna run dev --parallel",
     "dev:ui": "lerna run dev:ui --parallel",
-    "clean": "rm -rf packages/extension/dist packages/get-extension-wallet/dist",
+    "clean": "rm -rf packages/extension/dist packages/get-extension-wallet/dist packages/dapp/dist && rm -rf packages/extension/node_modules packages/get-extension-wallet/node_modules packages/dapp/node_modules",
```

Is there a need to repeat the `rm -rf` command?

```suggestion
    "clean": "rm -rf packages/extension/dist packages/get-extension-wallet/dist packages/dapp/dist packages/extension/node_modules packages/get-extension-wallet/node_modules packages/dapp/node_modules",
```

---

Nice, will update, thanks!